x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -55,33 +54,44 @@ export const LatestFindingsContainer = ({ dataView }: { dataView: DataView }) =>
         setQuery={setUrlQuery}
         query={urlQuery.query}
         filters={urlQuery.filters}
-        loading={findingsGroupByNone.isLoading}
+        loading={findingsGroupByNone.isFetching}
       />
       <PageWrapper>
-        <PageTitle>
-          <PageTitleText
-            title={
-              <FormattedMessage id="xpack.csp.findings.findingsTitle" defaultMessage="Findings" />
-            }
-          />
-        </PageTitle>
+        <LatestFindingsPageTitle />
         <FindingsGroupBySelector type="default" />
         <FindingsDistributionBar
           total={findingsGroupByNone.data?.total || 0}
           passed={findingsCount.data?.passed || 0}
           failed={findingsCount.data?.failed || 0}
-          pageStart={urlQuery.from + 1} // API index is 0, but UI is 1
-          pageEnd={urlQuery.from + urlQuery.size}
+          pageStart={urlQuery.pageIndex * urlQuery.pageSize + 1} // API index is 0, but UI is 1
+          pageEnd={urlQuery.pageIndex * urlQuery.pageSize + urlQuery.pageSize}
         />
         <EuiSpacer />
         <FindingsTable
-          {...urlQuery}
-          setQuery={setUrlQuery}
           data={findingsGroupByNone.data}
           error={findingsGroupByNone.error}
-          loading={findingsGroupByNone.isLoading}
+          loading={findingsGroupByNone.isFetching}
+          pagination={getPaginationTableParams({
+            pageSize: urlQuery.pageSize,
+            pageIndex: urlQuery.pageIndex,
+            totalItemCount: findingsGroupByNone.data?.total || 0,
+          })}
+          sorting={{
+            sort: { field: urlQuery.sort.field, direction: urlQuery.sort.direction },
+          }}
+          setTableOptions={({ page, sort }) =>
+            setUrlQuery({ pageIndex: page.index, pageSize: page.size, sort })
+          }
         />
       </PageWrapper>
     </div>
   );
 };
+
+const LatestFindingsPageTitle = () => (
+  <PageTitle>
+    <PageTitleText
+      title={<FormattedMessage id="xpack.csp.findings.findingsTitle" defaultMessage="Findings" />}
+    />
+  </PageTitle>
+);
```

Comment:
nit but I'm not sure why this static component needs to be separated. I would just leave it inside the page component

Reply:
probably subjective, but IMO it makes the root component read nicer as we don't look at long lines that aren't really interesting. `LatestFindingsPageTitle` tells me all i need to know without looking how it's done